libs/service-portal/health/src/screens/OrganDonation/OrganDonation.tsx

```diff
@@ -0,0 +1,70 @@
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { getOrganDonor } from '../../utils/OrganDonationMock'
+import {
+  ActionCard,
+  IntroHeader,
+  LinkResolver,
+} from '@island.is/service-portal/core'
+import { messages as m } from '../../lib/messages'
+import { Button, Box, Text } from '@island.is/island-ui/core'
+import { HealthPaths } from '../../lib/paths'
+import { Problem } from '@island.is/react-spa/shared'
+const OrganDonation = () => {
+  useNamespaces('sp.health')
+
+  const { formatMessage, lang } = useLocale()
+  const { data, loading, error } = getOrganDonor(lang)
+
+  return (
+    <Box>
+      <IntroHeader
+        title={formatMessage(m.organDonation)}
+        intro={formatMessage(m.organDonationDescription)}
+      />
+      {!error && !loading && data.data !== null && (
+        <>
+          <Box>
+            <LinkResolver
+              href={formatMessage(m.organDonationLink)}
+              key="organ-donation"
+            >
+              <Button
+                variant="utility"
+                size="small"
+                icon="open"
+                iconType="outline"
+              >
+                {formatMessage(m.readAboutOrganDonation)}
+              </Button>
+            </LinkResolver>
```

Comment:
Doesn't the `<LinkButton /> `do this?

Reply:
Ah might be, I didnt even think of it 🫨  I'll take a look

Reply:
Ill keep it like that, the button gets the whole width and i would have to add css class to fix